tasks/main.yml

```diff
@@ -35,10 +35,12 @@
       vars:
         failure: >-
           {%- for tunnel in vpn_connections | d([]) %}
-          {%   if tunnel.enc_type == 'cert' and tunnel.hosts.items()%}
-          {%     for key, value in tunnel.hosts.items() | d({})%}
-          {%       if value is none or 'cert_name' not in value -%}True{%-endif %}
-          {%     endfor %}
+          {%   if tunnel.enc_type == 'cert' %}
+          {%     if tunnel.hosts.items()%}
```

Comment:
I think you can just use `if tunnel.hosts` to test if it is empty - also you need a space before the `%}`

Reply:
I just changed this line in the last commit, it is now checking to make sure that `tunnel.hosts` is a mapping instead of using `items()`